modular_stonehedge/code/datums/traits/unspecial.dm

```diff
@@ -364,7 +363,7 @@
 /datum/quirk/unlucky
 	name = "Unlucky"
 	desc = "Ever since you knocked over that glass vase, you just feel... off"
-	value = -1
+	value = -6
```

Comment:
i don't know if -6 is justified but i also dont know if luck is that important.

Reply:
-6 points is fair for a trait that can leave you with anywhere from 1 to 10 luck. It might not affect a huge amount now, but it affects how often you find gems while mining.